pages/recipes/supabase.mdx

```diff
@@ -1,24 +1,157 @@
 import { Callout } from "nextra/components";
 
-# Supabase
+# Overview
+
+This page is broken down into two sections:
+
+- [Seeding a supabase local development stack](/recipes/supabase#seeding-a-supabase-local-development-stack)
+- [Restoring to a supabase local development stack](/recipes/supabase#restoring-to-a-supabase-local-development-stack)
+
+![I absolutely love Supabase!](/recipes/supabase/snappy-holding-supabase-logo.svg)
+
+## Seeding a supabase local development stack.
+
+Supabase makes it easy to set up a [local environment](https://supabase.com/docs/guides/cli/local-development) linked to your Supabase project. Snaplet can extend this functionality by helping you seed your local database. This ensures that you have readily available data to work with initially and for any future feature or testing needs.
+
+In this short guide, we will use the [nextjs + supabase template](https://vercel.com/templates/next.js/supabase), to demonstrate how one could use the `snaplet generate` command to seed their local supabase stack.
```

Comment:
Next.js + Supabase. Remember to always spell brand names the way they're spelled by the people who develop them.